.travis/install-libsodium.sh

```diff
@@ -24,7 +24,7 @@ version=1.0.11
 source_dir="$1"
 mkdir -p ${source_dir}
 pushd ${source_dir}
-wget https://download.libsodium.org/libsodium/releases/old/libsodium-${version}.tar.gz
+wget https://download.libsodium.org/libsodium/releases/old/unsupported/libsodium-${version}.tar.gz
```

Comment:
Looks like this change got lost.

Pull it out into its own commit, and open a new PR for it. I'll accept it as soon as you do, but then we'll have it better documented (since it's not specific to this tool PR).

Reply:
Sorry! PR has been opened.